### PR TITLE
refactor(send): remove old send flow

### PR DESCRIPTION
**Pull request by satish-ravi**
### Description

Changes include:
- Removing use_new_send_flow gate and all false code associated with it
- Removing all of the old send flow code and its dependencies (~1k lines flagged by knip)
- Updating secure send code to not take transactionData as parameter, which was only set in the old send flow
- Moving some helper functions in the old send flow that were also required in the new send flow to a more common location (E.g., resolveId, shared types, etc.)

Requires #4867, #4845 for e2e tests to pass

### Test plan

CI

### Related issues

- Fixes ACT-1075

### Backwards compatibility

Yes

### CODE REVIEW

File: src/send/SendConfirmation.test.tsx [hunk 1]
```diff
@@ -184,29 +132,12 @@ describe('SendConfirmation', () => {
...
-    })
-
-    expect(getByTestId('feeDrawer/SendConfirmation/totalFee')).toHaveTextContent('₱0.027')
-
-    // Subtotal is $1.33, which is added the fee amount.
-    const totalComponent = getByTestId('TotalLineItem/Total')
-    expect(getElementText(totalComponent)).toEqual('₱1.36')
-  })
-
   it('renders correctly for send payment confirmation with fees from props (new UI)', async () => {
```
Comment (cajubelt) [inline src/send/SendConfirmation.test.tsx:139]: should probly remove "new UI" part of this (won't age well)

Comment (cajubelt) [reply src/send/SendConfirmation.test.tsx:139]:
```suggestion
  it('renders correctly for send payment confirmation with fees from props', async () => {
```

**Review by cajubelt — APPROVED** [review]
massive improvement 🙏

**Review by jh2oman — APPROVED** [review]
Looks amazing!

**Pull request merged**